index.html

```diff
@@ -0,0 +1,174 @@
+        <p>To get started with writing JavaScript, open the Scratchpad and write your first "Hello world" JavaScript code:</p><br>
+        <code class="graybackground">
+          <p>To get started with writing JavaScript, open the Scratchpad and write your first "Hello world" JavaScript code:</p>
+          <p>
+            greetMe("World");
+          </p>
+        </code>
+
+      </section>
+      <section id="variables" class="main-section">
+        
+        <header ><h1>Variables</h1></header>
+        <p>You use variables as symbolic names for values in your application. The names of variables, called identifiers, conform to certain rules.<br>
+
+          A JavaScript identifier must start with a letter, underscore (_), or dollar sign ($); subsequent characters can also be digits (0-9). Because JavaScript is case sensitive, letters include the characters "A" through "Z" (uppercase) and the characters "a" through "z" (lowercase).<br>
+          
+          You can use ISO 8859-1 or Unicode letters such as å and ü in identifiers. You can also use the Unicode escape sequences as characters in identifiers. Some examples of legal names are Number_hits, temp99, and _name.</p>
+
+      </section>
+      <section id="Declaring_variables" class="main-section">
+        <header ><h1>Declaring variables</h1></header>
+        <p>
+          You can declare a variable in three ways:<br>
+          With the keyword var. For example, <br>
+          <code class="graybackground">
+            var x = 42;
+          </code><br>
+          This syntax can be used to declare both local and global variables.<br>
+          By simply assigning it a value. For example,<br>
+          <code class="graybackground">
+            x = 42.
+          </code><br>
+          This always declares a global variable. It generates a strict JavaScript warning. You shouldn't use this variant.<br>
+          With the keyword let. For example,<br>
+          <code class="graybackground">
+            let y = 13.
+          </code><br>
+          This syntax can be used to declare a block scope local variable. See Variable scope below.
+        </p>
+
+      </section>
+      <section id="Variable_scope" class="main-section">
+        <header ><h1>Variable scope</h1></header>
+        <p>
+          When you declare a variable outside of any function, it is called a global variable, because it is available to any other code in the current document. When you declare a variable within a function, it is called a local variable, because it is available only within that function.<br>
+          JavaScript before ECMAScript 2015 does not have block statement scope; rather, a variable declared within a block is local to the function (or global scope) that the block resides within. For example the following code will log 5, because the scope of x is the function (or global context) within which x is declared, not the block, which in this case is an if statement.<br>
```

Comment:
It is not considered best practice to right more than 100 characters in the same line. Please fix this.